File: commons/src/main/java/it/uniroma1/commons/queue/object/Detection.java
package it.uniroma1.commons.queue.object;

import it.uniroma1.commons.queue.enums.CarType;
import it.uniroma1.commons.queue.enums.FuelType;
import it.uniroma1.commons.queue.enums.RoadType;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
@Getter
@Setter
@ToString
@NoArgsConstructor

public class Detection {

    private Long id;

    private Long speedCameraId;

    private int speedValue;

    private String licensePlate;

    private String region;

    private RoadType roadType;
}

File: statistics/src/main/java/javastatistics/Detection.scala
package javastatistics
import it.uniroma1.commons.queue.enums.{CarType, FuelType, RoadType}

class Detection(val id: Long, val speedCameraId: Long, val speedValue: Int,
                val licensePlate: String, val carType: CarType, val fuelType: FuelType,
                val region: String, val roadType: RoadType, val overcameLimit: Boolean)
